sync/handler_hello_test.go

```diff
@@ -44,8 +45,24 @@ func TestParsingHelloMessages(t *testing.T) {
 
 			assert.NoError(t, td.receivingNewMessage(td.sync, msg, pid))
 			td.checkPeerStatus(t, pid, peerset.StatusCodeBanned)
-			bundle := td.shouldPublishMessageWithThisType(t, td.network, message.TypeHelloAck)
-			assert.Equal(t, bundle.Message.(*message.HelloAckMessage).ResponseCode, message.ResponseCodeRejected)
+			bdl := td.shouldPublishMessageWithThisType(t, td.network, message.TypeHelloAck)
+			assert.Equal(t, bdl.Message.(*message.HelloAckMessage).ResponseCode, message.ResponseCodeRejected)
+		})
+
+	t.Run("Receiving Hello message from a peer. Difference of Hello Message Time with node time is a lot.",
```

Comment:
We need two tests:
In the first test, `my_time` is: `Now() - 10 seconds`.
In the second test, `my_time` is: `Now() + 20 seconds`.

I purposely added an extra 10 seconds to the time to support running tests in slow CI/CD actions.